packages/react-app-revamp/components/Onramp/providers/coinbase/index.tsx

```diff
@@ -20,7 +20,8 @@ const COINBASE_PARAMS = {
 const OnrampCoinbaseProvider: FC<OnrampCoinbaseProviderProps> = ({ chain, asset }) => {
   const isEntryPage = usePathname().includes("submission");
   const { address } = useAccount();
-  const isSupported = isChainSupported(chain);
+  // TODO: remove this once we have a new logic for coinbase (session token etc.)
```

Comment:
just wanted to check in on this note/logic since we're just planning to deprecate